### PR TITLE
spirv-fuzz: Edit TransformationLoad structure to supporting OpLoad and OpAtomicLoad together.

### PR DESCRIPTION
**Pull request by Mostafa-ashraf19**
This RR is just to support the "TransformationLoad" new structure. 
Part of #4324.

### CODE REVIEW

**Comment by Mostafa-ashraf19** [comment]
@afd  ping. 

when you accept changes I will merge this branch with the main branch I worked on it. What do you think?

**Review by afd — CHANGES_REQUESTED** [review]
It turned out I had already suggested some changes on this but had forgotten to hit submit :)

File: source/fuzz/protobufs/spvtoolsfuzz.proto [hunk 1]
```diff
@@ -1562,15 +1562,24 @@ message TransformationLoad {
...
   uint32 fresh_id = 1;
 
-  // The pointer to be loaded from
+  // The pointer to be loaded from.
   uint32 pointer_id = 2;
 
+  // Flag to check current load opCode is atomic or not, to take action for each one.
+  bool is_atomic = 3;
+
+  // Define the scope of execution.
```
[the remark this answers is not in the document]
Comment (Mostafa-ashraf19) [reply source/fuzz/protobufs/spvtoolsfuzz.proto:1574]: Done.

File: source/fuzz/protobufs/spvtoolsfuzz.proto [hunk 2]
```diff
@@ -1562,15 +1562,24 @@ message TransformationLoad {
...
 
-  // The result of the load instruction
+  // The result of the load instruction.
   uint32 fresh_id = 1;
 
-  // The pointer to be loaded from
+  // The pointer to be loaded from.
   uint32 pointer_id = 2;
 
+  // Flag to check current load opCode is atomic or not, to take action for each one.
```
[the remark this answers is not in the document]
Comment (Mostafa-ashraf19) [reply source/fuzz/protobufs/spvtoolsfuzz.proto:1571]: Done.

**Pull request closed without merging**